### ISSUE TITLE
Missing icon in Windows Terminal profile

### ISSUE THREAD

**Issue by dact221** [JuliaLang/juliaup#160]
Dear developers,

I installed Julia via Microsoft Store (Windows 11), and the package installed a profile for Windows Terminal. However, the Terminal profile is missing the Julia icon:

![image](https://user-images.githubusercontent.com/35376194/142742329-6286d737-151b-4e4f-9e3b-4b651009a472.png)

I checked the URI in the profile and _apparently_ it's ok:

```
ms-appx://JuliaComputingInc.Julia/Public/Fragments/julia.ico
```

However, I read the [Windows App Development docs](https://docs.microsoft.com/en-us/windows/uwp/app-resources/uri-schemes) and found that the The package name in the `ms-appx` URI scheme must be the name of one of the packages **in the current running app's package dependency graph** (i.e., Terminal dependencies).

According to that, it seems that it's not possible for Terminal to access the Julia icon through URI schemes. I have been thinking of another way that doesn't require the user to input a custom path, but nothing comes to my mind.

**Comment by davidanthoff** [comment]
Yes, unfortunately that is an upstream problem, see [here](https://github.com/microsoft/terminal/issues/10359).

**Comment by claudeLovelace** [comment]
This is still an issue.

**Comment by b4shful** [comment]
@davidanthoff Upstream issue has been seemingly fixed and is now closed. Although the issue with the Julia icon still persists.

**Comment by diegozea** [comment]
Hi! I just installed the latest version of Julia and Windows Terminal on Windows 11, but the Julia icon is missing.

**Comment by davidanthoff** [comment]
It works correctly in the Windows Terminal Preview build, so we just have to wait until the next release of Windows Terminal ships, and then the correct icon should show up :) I'm closing this issue, nothing for us to do.
